visual-js/visual-snapshots/setEnvVars.ts

```diff
@@ -0,0 +1 @@
+process.env.EXISTING_ENV_VARIABLE= 'existing-env-variable-value';
```

Comment:
It seems like you're using this only in `params.spec.ts`. Why not add this to a `beforeAll` hook?

Reply:
The file is redundant after setting the env in the test file